ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -27,6 +27,7 @@ Item {
     property bool isContact: false
     property bool isCurrentUser
     property bool userIsEnsVerified
+    property rect cropRect: undefined
```

Comment:
Just keep it empty; defining to `undefined` is kinda strange

Reply:
@caybro yes, I thought the same, but I based on instructions for Qt5.15 where `Image` has `sourceClipRect`:
```
This property can be cleared to reload the entire image by setting sourceClipRect to undefined.
```

So I did it for a future :) if you think it's no needed I can fix it.

Reply:
Good idea!